lib/curl_option.ml

```diff
@@ -0,0 +1,32 @@
+open Astring
+
+type auth = { user : string; token : string }
+
+type t =
+  | Location
+  | User of auth
+  | Silent
+  | Show_error
+  | Config of [ `Stdin | `File of string ]
+  | Dump_header of [ `Ignore | `File of string ]
+  | Data of [ `Data of string | `File of string ]
+  | Data_binary of [ `Data of string | `File of string ]
+  | Header of string
+
+let to_string_list opts =
+  List.fold_left
+    (fun acc -> function Location -> "--location" :: acc
+      | User { user; token } -> "--user" :: strf "%s:%s" user token :: acc
+      | Silent -> "--silent" :: acc | Show_error -> "--show-error" :: acc
```

Comment:
Really not important at all, but I was wondering: is it intentional here to have two cases in one line?

Reply:
It's the result of using the `break-cases= fit` option of ocamlformat (conventional behavior), I agree it doesn't look good here but most of the time it works well

Reply:
Cool, thanks for the explanation!